metron-interface/metron-alerts/src/app/alerts/alerts-list/table-view/table-view.component.spec.ts

```diff
@@ -31,12 +31,20 @@ import { UpdateService } from '../../../service/update.service';
 import { GlobalConfigService } from '../../../service/global-config.service';
 import { MetaAlertService } from '../../../service/meta-alert.service';
 import { DialogService } from 'app/service/dialog.service';
+import {AppConfigService} from "../../../service/app-config.service";
```

Comment:
If possible, could you use single quotes and add spacing inside the brackets? This formatting is specified by our tslint config, and editors like VS code will show an error if it's not fixed.